nbviewer/formats.py

```diff
@@ -45,7 +45,7 @@ def default_formats():
             'nbconvert_template': 'slides_reveal',
             'label': 'Slides',
             'icon': 'gift',
-            'test': lambda nb, json: '"slideshow"' in json,
+            'test': lambda nb, json: 'celltoolbar' in nb.metadata and nb.metadata['celltoolbar'] == 'Slideshow',
```

Comment:
Does `'celltoolbar' in nb.metadata` only evaluate to true if the notebook author had the slide toolbar visible when they last saved the notebook? Is that the intended behavior change here?